SU-HSDO/suhumsci
# Arch 56 default footer blocks

# READY FOR REVIEW

# Summary
- This depends on `ARCH-58-basic-page`. it can be done before or after reviewing `ARCH-58-basic-page` but if its after, the base branch will have to change.
- Create 4 default blocks that are placed in the footer on install.

# Needed By (Date)
- Whenever

# Urgency
- Low

# Steps to Test

1. Checkout this branch and run composer update/install
2. install fresh site
3. verify the 4 default blocks display on the home page at the bottom.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
